src/java/com/zimbra/cs/taglib/tag/LoginTag.java

```diff
@@ -266,4 +289,29 @@ public static void setTrustedCookie(HttpServletResponse response, String trusted
             ZimbraCookie.clearCookie(response, name);
         }
     }
+
+    public static boolean isCaptchaValid(String captchaId, String captchaInput) {
+        try {
+            String url = "http://localhost:8666/verifyCaptcha?"
```

Comment:
Comments:
1. Should be using TLS for all service accesses, even internal services
2. Does this URL/service need to be configurable?